README.md

```diff
@@ -112,6 +112,7 @@ Note that building isn't necessary for end users and is more for developers look
 
 ```sh
     git clone git@github.com:IBM/nodejs-idb-connector.git
+    yum install sqlcli-devel
```

Comment:
I'd rather have the yum command first as it interjects between cloning the repo and `cd`ing in to it.